conf/resources/data/GB000170/19GB0000601001FBD8/response.json

```diff
@@ -0,0 +1,5 @@
+{
+  "MRN": "19GB0000601001FBD8",
+  "ResponseCode":3,
+  "ErrorDescription":"Not found' (mimics situation where code should be GB000060)"
```

Comment:
This should be: 
'Unknown MRN' as per the error responses we are expecting to get back here: https://confluence.tools.tax.service.gov.uk/display/CoTCo/CTC+3.0+NCTS+NCF+API